modules/rest-api/pages/rest-configure-saml.adoc

```diff
@@ -0,0 +1,205 @@
+
+|spContactName
+|The name of the administrator responsible for Couchbase Server's SAML integration.
+
+|spEntityId
+|The identifier Couchbase Server uses when exchanging SAML messages with the IdP. It defaults to the URL for Couchbase Server's SAML metadata. The actual content of the field is not important as long as the IdP recognizes the identifier.
+
+|spKey
+|The private key Couchbase Server uses to sign and encrypt SAML messages in PEM format. You must supply a private key to enable SAML authentication. This parameter is write-only. A GET request only returns a string of asterisks for this value. 
+
+|spLogoutURL
+|The URL the IdP uses to log the user out of the Web Console. By default, Couchbase Server sets this value based on spBaseURLType and spBaseURLScheme.
+
+|spMetadataCacheDuration
+|Sets how long the IdP caches Couchbase Server's metadata. This value is in https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO8601 duration format^]. The default value `P1M` sets this duration to one month.
+
+|spMetadataURL
+|The URL to retrieve Couchbase Server's SAML metadata. By default, Couchbase Server sets this value based on spBaseURLType and spBaseURLScheme.
+
+|spOrgDisplayName
+|An alternate name of your organization. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spOrgName
+|The name of your organization. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spOrgURL
+|A URL for your organization. This value does not play a role in authentication. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spSessionExpire
+|String value that controls whether the Web Console expires its session when the IdP's authorization expires. When set to the default `SessionNotOnOrAfter` value, the Web Console has the user reauthenticate when the IdP's authentication expires. If set to `false`, the Web Console does not set its session length to match the IdP's authentication expiration.
+
+|spSignMetadata
+|A Boolean value that controls whether Couchbase Server signs its metadata using the key set in spKey. Defaults to `true`, which enables signing the metadata.
+
+|spSignRequests
+|A Boolean value that controls whether Couchbase Server signs its SAML requests to the IdP. Defaults to `true` which enables signing the requests.
+
+|spTrustedFingerprints
+|One or more certificate fingerprints that Couchbase Server can use to authenticate messages from the IdP instead of using certificates. 
+
+|spTrustedFingerprintsUsage
+|String value that controls what Couchbase Server verifies using certificate fingerprints. When set to the default `metadataInitialOnly` value, it uses fingerprints to verify just the metadata it retrieves from the IdP. If set to `everything` it uses fingerprints for all SAML message verification. Only has an effect if at least one fingerprint is set in spTrustedFingerprints.
```

Comment:
There are three values:
metadataOnly - it uses fingerprints  to verify signatures the IdP metadata signature it retrieves from the IdP (everytime it retrieves it)
metadataInitialOnly - it uses fingerprints  to verify the IdP metadata signature on first retrieval only (bootstrap), but after that, it uses certificates provided by IdP in IdP metadata for signature validation.
everything - it uses fingerprints for all SAML message signature verification (completely ignores certificates it receives in IdP metadata).